src/app/search/map-friendly-values.pipe.ts

```diff
@@ -103,7 +103,9 @@ export class MapFriendlyValuesPipe implements PipeTransform {
         [SourceFile.TypeEnum.DOCKSTOREYML, 'Configuration'],
         [SourceFile.TypeEnum.DOCKSTORESERVICEOTHER, 'Service Files']
       ])
-    ]
+    ],
+    ['success', new Map([['true', 'Success'], ['false', 'Failed']])],
```

Comment:
though I guess it doesn't matter too much, does the filtering still work for these columns intuitively?

Reply:
Filtering doesn't work on the new map-friendly values, but currently working on a method to fix that issue.

Reply:
Fixed. Though the filter still works with the original values, along with the new mapped values.